common/setups/rasr/base_decoder.py

```diff
@@ -77,6 +77,11 @@ def __init__(
             "mfcc": features.MfccJob,
         }
 
+        # holds the recognition jobs: search, lat2ctm, score, optlm
+        # Dict[str, Dict[str, Dict[str, Type[tk.Job]]]]
+        # self.jobs[CORPUS_KEY][JOB_TYPE][EXP_NAME] = tk.Job
+        self.jobs = {}
```

Comment:
```suggestion
        self.jobs:  Dict[str, Dict[str, Dict[str, Type[tk.Job]]]] = {}
```
I think this should work or you could do `self.jobs = {} # type: Dict[str, Dict[str, Dict[str, Type[tk.Job]]]]`